### PR TITLE
chore: fix typo

### PR DESCRIPTION
**Pull request by hezhizhen**
Signed-off-by: Zhizhen He <hezhizhen.yi@gmail.com>

### CODE REVIEW

File: docs/api/config.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Configure jupyter notebook configuration
 
 **Arguments**:
 
-- `password` _str_ - Password for access authenticatioin
+- `password` _str_ - Password for access authentication
```
Comment (gaocegege) [inline docs/api/config.md:16]: Thanks for the fix!

The file is generated from https://github.com/tensorchord/envd/blob/main/envd/api/config/__init__.py#L49

Would you mind filing a new PR to fix there?

**Review by gaocegege — APPROVED** [review]
Thanks for your contribution! :tada: :+1:

File: docs/api/config.md (hunk 1 above)
Comment (hezhizhen) [reply docs/api/config.md:16]: later tonight

**Comment by hezhizhen** [comment]
https://github.com/tensorchord/envd/pull/500

**Pull request merged**